devito/operator.py

```diff
@@ -201,7 +201,9 @@ def _dle_arguments(self, dim_sizes):
         dle_arguments = OrderedDict()
         autotune = True
         for i in self.dle_arguments:
-            dim_size = dim_sizes.get(i.original_dim.name, i.original_dim.size)
+            dim_size = dim_sizes.get(i.original_dim.name,
+                                     i.original_dim.size if i.original_dim.is_Fixed
```

Comment:
What do you think of adding a ``size`` property to the "main" ``Dimension`` class, returning ``None``, so that we don't need ``if ... else`` any time we check for the presence of a ``FixedDimension``?

Reply:
Could that wait for a future cleanup or do you want it in this PR?

Reply:
Can wait